libs/ui/elements/src/lib/metadata-info/metadata-info.component.html

```diff
@@ -32,27 +35,30 @@
   *ngIf="metadata.lineage"
   [title]="'record.metadata.origin' | translate"
 >
-  <p class="mb-5 pt-4 whitespace-pre-line break-words" gnUiLinkify>
+  <p
+    class="text-gray-900 mb-5 pt-4 whitespace-pre-line break-words"
```

Comment:
I thought, we would let the config set the `main` color here (as well as in the abstract above), but maybe I'm wrong (cc @fgravin @jahow)